linkerd/proxy/http/src/lib.rs

```diff
@@ -50,6 +48,7 @@ pub use http::{
 pub use hyper::body::HttpBody;
 pub use linkerd_http_box::{BoxBody, BoxRequest, BoxResponse, EraseResponse};
 pub use linkerd_http_executor::TracingExecutor;
+pub use linkerd_http_version::{self as version, Version};
```

Comment:
reëxports are added to make this a backwards compatible change, i.e. no
changes to the public api of linkerd-proxy-http are performed here.